src/react-vite/10-nested-routes/nested-routes.md

```diff
@@ -18,7 +18,7 @@ In this section, we will:
 
 We want to have a component to display individual restaurants details, and want the path to be nested under the restaurants path.
 
-<img alt="Screenshot of the Place My Order app with a nested route, displaying the “/restaurants/lettuce-shack” URL in the browser’s address bar." src="../static/img/react-vite/10-nested-routes/01-solution.png" style="max-width: 735px;"/>
+<img alt="Screenshot of the Place My Order app with a nested route, displaying the “/restaurants/lettuce-shack” URL in the browser’s address bar." src="../static/img/react-vite/10-nested-routes/01-solution.png" style="max-width: 640px;"/>
```

Comment:
What’s this value based on? I just checked the image and it should be a max of 735px wide.

Reply:
735px image was overlapping with the side bar. 640px was used in `/src/react-vite/03-intro-to-jsx/intro-to-jsx.md` so I went with that.

Reply:
Let’s move these values out of the `style` attribute and specify the `height` and `width` attributes on the image: https://bitovi.atlassian.net/browse/LD-204